pytest-selenium/pages/base.py

```diff
@@ -0,0 +1,24 @@
+from time import sleep
+
+import pypom
+from selenium.webdriver.common.keys import Keys
+
+
+class Page(pypom.Page):
```

Comment:
the reason we're using python in the first place is because mike has already solved this problem using python and doesn't have to re-invent everything to set up rex in the same way. down the line once we get things working i'd consider porting the suite over to typescript. in the meantime i don't think its worth going too far out of our way to modify the base pytest framework (like adding types)

Reply:
it will be some time before I can go through and add all the type hints.